db/migrations/mysql/000026_create_oauth_apps.up.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS OAuthApps (
+  Id varchar(26) NOT NULL,
+  CreatorId varchar(26) DEFAULT NULL,
+  CreateAt bigint(20) DEFAULT NULL,
+  UpdateAt bigint(20) DEFAULT NULL,
+  ClientSecret varchar(128) DEFAULT NULL,
+  Name varchar(64) DEFAULT NULL,
+  Description text,
+  CallbackUrls text,
+  Homepage text,
+  PRIMARY KEY (Id),
+  KEY idx_oauthapps_creator_id (CreatorId)
```

Comment:
We should make this a `create if not exists`. This will be skipped if the table exists but the index doesn't. Something like:

```sql
SET @preparedStatement = (SELECT IF(
    (
        SELECT COUNT(*) FROM INFORMATION_SCHEMA.statistics
        WHERE table_name = 'OAuthApps'
        AND table_schema = DATABASE()
        AND index_name = 'idx_oauthapps_creator_id'
    ) > 0,
    'SELECT 1',
    'CREATE INDEX ON OAuthApps(CreatorId);'
));

PREPARE createIndexIfNotExists FROM @preparedStatement;
EXECUTE createIndexIfNotExists;
DEALLOCATE PREPARE createIndexIfNotExists;
```